### ISSUE TITLE
update documentation install steps

### ISSUE THREAD

**Issue by brianbaker** [markitondemand/moment-msdate#4]
Documentation should be updated to instruct users to run `npm install --save moment-msdate`.  Also, see about momentjs updating their documentation located here: http://momentjs.com/docs/#/plugins/msdate/

**Comment by markhealey** [comment]
When I originally put this together the momentjs site was simply forked and
PR'd back into their repo.

**Comment by markhealey** [comment]
Yes @brianbaker, see [this](https://github.com/markitondemand/momentjs.com/commits/07f62b5b51eb0789132f81a59f402e6e419e806e/source/core_docs/plugins/msdate.md) for reference. Hope all's well!
